server/views/premises/show.njk

```diff
@@ -28,9 +28,9 @@
       },
       menus: [{
         items: [ {
-          text: "Manage rooms",
+          text: "Manage beds",
```

Comment:
The commit message is a little confusing for this change. 
Should it be 'We’re now acting directly on beds, not rooms'?